src/test/scala/org/broadinstitute/hail/driver/GRMSuite.scala

```diff
@@ -133,7 +132,9 @@ class GRMSuite extends SparkSuite {
     val grmBinFile = tmpDir.createTempFile("test", ".grm.bin")
     val grmNBinFile = tmpDir.createTempFile("test", ".grm.N.bin")
 
-    Prop.check(forAll(VariantSampleMatrix.gen[Genotype](sc, (v: Variant) => Genotype.gen(v).filter(_.isCalled))
+    Prop.check(forAll(VariantSampleMatrix.gen[Genotype](sc,
+      VSMSubGens(tGen = (v: Variant) => Genotype.genRealistic(v).filter(_.isCalled),
```

Comment:
I thought instead of doing VSMSubGens with default arguments, you'd have a few predefined VSMSubGen clients could use.  Then you could write `VSMSubGens.plink.copy(tGen = VSMSubGens.plink.tGen.filter(_.isCalled))`.  This also seems fine.  No need to change anything.  Just thought I'd throw out the alternative.